bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3747,57 +3746,29 @@ void ReplicaImp::executeRequestsInPrePrepareMsg(concordUtils::SpanWrapper &paren
       LOG_DEBUG(CNSUS, "Consensus reached");
     }
     SCOPED_MDC("pp_msg_cid", ppMsg->getCid());
+    std::deque<IRequestsHandler::ExecutionRequest> accumulatedRequests;
```

Comment:
use the typedef - `IRequestsHandler::ExecutionRequestsQueue`

Reply:
Fixed.